config.py

```diff
@@ -60,6 +60,7 @@ class Config(object):
     RESPONSE_OPERATIONS_UI_HOST = os.getenv('RESPONSE_OPERATIONS_UI_HOST', 'localhost')
     RESPONSE_OPERATIONS_UI_PORT = os.getenv('RESPONSE_OPERATIONS_UI_PORT', 8085)
     RESPONSE_OPERATIONS_UI = '{}://{}:{}'.format(PROTOCOL, RESPONSE_OPERATIONS_UI_HOST, RESPONSE_OPERATIONS_UI_PORT)
+    RESPONSE_OPERATIONS_UI_COLLECTION_EXERCISE = '{}/surveys/{}'.format(RESPONSE_OPERATIONS_UI, '{}')
```

Comment:
We should probably just put this in `collection_exercise.py` as it's not really configuration as you can't override it and then we only need to format it once